js/web/script/parse-profiler.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+
+// parse-profiler
+//
+// this script is used to parse performance profiling result.
+// usage:
+// STEP.1 - profiling
+// > npm test -- model test/test-data/{path-to-my-model} --backend={cpu/webgl/wasm} --profile > profile.raw.log
+// STEP.2 - parse
+// > node tools/parse-profiler < profile.raw.log > profile.parsed.log
+
+
+import * as readline from 'readline';
+const int = readline.createInterface({input: process.stdin, output: process.stdout, terminal: false});
```

Comment:
change name 'int' to better name like 'lines' or 'profileData'

Reply:
done.